### DISCUSSION TITLE
auto-install-peers=true ?

### DISCUSSION THREAD

**Discussion by stephenjason89** [nuxt/nuxt#18538] [Questions]
Using nuxt 3 (pnpm) with eslint throws me multiple missing peer errors
<img width="1225" alt="image" src="https://user-images.githubusercontent.com/3810177/214784369-4bf9a994-d3ea-4562-89b1-b7b3dee32884.png">

Should I use auto-install-peers? What's the best way to deal with these errors?

**Comment by TheAlexLichter** [comment]
No, you don't need to. You can usually turn off the warnings for missing peers.

**Reply by stephenjason89** [reply]
Hello @manniL , is strict-peer-dependencies: false is what you're referring to?

**Reply by TheAlexLichter** [reply]
Yes, exactly.

**Comment by stephenjason89** [comment]
@manniL after adding strict-peer-dependencies on my .npmrc
<img width="306" alt="image" src="https://user-images.githubusercontent.com/3810177/219968618-be0421ae-544e-4316-b3cb-6993e2da6175.png">

i still get
<img width="1461" alt="image" src="https://user-images.githubusercontent.com/3810177/219968594-90ee0629-79eb-4a7a-9c6a-4cce60a5f92a.png">
